batcher/aligned-batcher/src/lib.rs

```diff
@@ -566,19 +575,77 @@ impl Batcher {
             .map(|(i, signature)| SignatureData::new(signature, nonces[i]))
             .collect();
 
-        eth::create_new_task(
-            payment_service,
-            *batch_merkle_root,
+        match self
+            .create_new_task(
+                *batch_merkle_root,
+                batch_data_pointer,
+                leaves,
+                signatures,
+                AGGREGATOR_COST.into(), // FIXME(uri): This value should be read from aligned_layer/contracts/script/deploy/config/devnet/batcher-payment-service.devnet.config.json
+                gas_per_proof.into(), //FIXME(uri): This value should be read from aligned_layer/contracts/script/deploy/config/devnet/batcher-payment-service.devnet.config.json
```

Comment:
remove these FIXME comments, they do not apply anymore

Reply:
removed on e45d40438dc3271f447adb9c402aa4b261e9f757